galaxy/templates/hapostgres/pgcluster.yaml

```
@@ -8,6 +8,9 @@ metadata:
   namespace: {{ .Release.Namespace }}
 spec:
   instances: {{ default "1" .Values.postgresql.instances }}
+  {{- if or (not .Values.postgresql.instances) (eq (toString .Values.postgresql.instances) "1") }}
```

If we specify .Values.postgresql.instances in the values file, could this then be simplified as follows?
```suggestion
  {{- if eq .Values.postgresql.instances 1 }}
```